src/main/perl/VXVM.pod

```diff
@@ -0,0 +1,7 @@
+=pod
```

Comment:
better leave the file empty, so it's clear there nothing in it (but a file has to be present afaik)